projects/client/src/app/components/footer/Footer.tsx

```diff
@@ -23,43 +45,81 @@ function Footer() {
 
         <div className='mb-4 flex flex-col items-start lg:mb-0 lg:items-start'>
           <p style={{ fontWeight: 'bold' }}>Useful Links</p>
-          <Button className='text-slate-500'>About</Button>
-          <Button className='text-slate-500'>Contact</Button>
-          <Button className='text-slate-500'>Wishlist</Button>
-          <Button className='text-slate-500'>Terms & Conditions</Button>
+          <Button
+            onClick={() => {
```

Comment:
hmmm destructu the onclick function:

` onClick={() => navigate('/about')}
  className='text-slate-500'`

Reply:
maybe you can reuse the class name, it is repeated more than one time